Thinktecture.Relay.OnPremiseConnector/SignalR/RelayServerConnection.cs

```diff
@@ -309,11 +306,20 @@ private void OnReconnecting()
 			}
 		}
 
-		private Task AcknowlegdeRequest(IOnPremiseTargetRequest request)
+		private Task AcknowlegdeRequestIfRequired(IOnPremiseTargetRequest request)
 		{
-			_logger?.Debug("Sending acknowlegde to relay server. request-id={RequestId}, acknowledge-id={AcknowledgeId}", request.RequestId, request.AcknowledgeId);
+			_logger.Verbose("Checking if acknowledged is required. request-id={RequestId}, acknowledgment-mode={AcknowledgmentMode}, acknowledge-id={AcknowledgeId}", request.RequestId, request.AcknowledgmentMode, request.AcknowledgeId);
```

Comment:
Maybe this isn't really necessary to log?

Reply:
That was very important during testing, so verbose is fine imho?